LICENSE

```diff
@@ -380,3 +384,377 @@ domain:
   - CompactConcurrentHashSet2, derived from JSR-166 ConcurrentHashMap v1.43
     (http://gee.cs.oswego.edu/dl/concurrency-interest).
   - tooltip.js v1.2.6 (https://github.com/jquerytools/jquerytools)
+
```

Comment:
It's not clear to me whether this CDDL 1.1 License belongs here in the top LICENSE or in the geode-assembly/src/main/dist/LICENSE which already has a CDDL 1.0 license within that gets appended to the top level license during assembly. Do we need both 1.0 and 1.1 for CDDL?